php/Terminus/CommandWithSSH.php

```diff
@@ -9,6 +9,33 @@
  * Base class for Terminus commands that deal with sending SSH commands
  */
 abstract class CommandWithSSH extends TerminusCommand {
+  /**
+   * A hash of commands which do not work in Terminus
+   * The key is the drush command
+   * The value is the Terminus equivalent, or null if DNE
+   */
+  protected $unavailable_commands = array();
+
+  /**
+   * Checks to see if the command is not available in Terminus and, if not,
+   * it will refer you to an equivalent Terminus command, if such exists.
+   *
+   * @param [string] $command The command to check for availability
+   * @return [void]
+   */
+  protected function checkCommand($command) {
```

Comment:
We're gonna run into trouble with the concrete implmentation when we try to implement this for WP-CLI too, but looks good for now: https://github.com/pantheon-systems/cli/wiki/Known-Issues#non-functional-terminus-wp-commands